<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Relo</class>
 <widget class="QMainWindow" name="Relo">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>936</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relocation Ticket</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <widget class="QGroupBox" name="Ticket_Buttons">
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QPushButton" name="Copy">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Copy to Clipboard</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Generate Ticket</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="Clear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>810</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>373</width>
         <height>791</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="CUSTOMER_INFORMATION">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Customer Information:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Disconnect:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_cstAddress_D">
          <property name="text">
           <string>Physical Address</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLineEdit" name="cstAddress_D"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_cstCity_D">
          <property name="text">
           <string>City, State Zip</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLineEdit" name="cstCity_D"/>
        </item>
        <item row="14" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Reinstall:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_cstAddress_R">
          <property name="text">
           <string>Physical Address (Reinstall)</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLineEdit" name="cstAddress_R"/>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="label_cstCity_R">
          <property name="text">
           <string>City, State Zip (Reinstall)</string>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLineEdit" name="cstCity_R"/>
        </item>
        <item row="22" column="0">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="23" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline; color:#aa0000;&quot;&gt;IMPORTANT:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="24" column="0">
         <widget class="QCheckBox" name="tktQHolder">
          <property name="text">
           <string>Confirmed account holder will be onsite for appointment.</string>
          </property>
         </widget>
        </item>
        <item row="25" column="0">
         <widget class="QCheckBox" name="tktQConfirm">
          <property name="text">
           <string>Customer is aware that we need a confirmation before heading out.</string>
          </property>
         </widget>
        </item>
        <item row="26" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Any animals onsite must be contained while our technician is on site.</string>
          </property>
         </widget>
        </item>
        <item row="27" column="0">
         <widget class="QCheckBox" name="tktQDisclose">
          <property name="text">
           <string>Disclosure Read.</string>
          </property>
         </widget>
        </item>
        <item row="40" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="28" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Relocation-Disconnect job has been created.</string>
          </property>
         </widget>
        </item>
        <item row="29" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Relocation-Installation job has been created.</string>
          </property>
         </widget>
        </item>
        <item row="21" column="0">
         <widget class="QComboBox" name="tktTower">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="20" column="0">
         <widget class="QLabel" name="towerBox">
          <property name="text">
           <string>Closest Tower to Reinstall:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QComboBox" name="tktZone">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Disconnect Zone:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QTextEdit" name="ticketTemplate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>810</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-54</y>
         <width>351</width>
         <height>907</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="ticketInfo">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Ticket Information:           &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Disconnect Date</string>
          </property>
         </widget>
        </item>
        <item row="20" column="0">
         <widget class="QCalendarWidget" name="tktCalendar_D">
          <property name="gridVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="21" column="0">
         <widget class="QDateEdit" name="tktDateText_D"/>
        </item>
        <item row="23" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="24" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Reinstall Date</string>
          </property>
         </widget>
        </item>
        <item row="25" column="0">
         <widget class="QCalendarWidget" name="tktCalendar_R"/>
        </item>
        <item row="26" column="0">
         <widget class="QDateEdit" name="tktDateText_R"/>
        </item>
        <item row="27" column="0">
         <widget class="QGroupBox" name="tkt_Group_TimeBlock">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>75</height>
           </size>
          </property>
          <property name="title">
           <string>Time Block Assigned</string>
          </property>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QRadioButton" name="tkt_am">
             <property name="text">
              <string>A.M.</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="tkt_pm">
             <property name="text">
              <string>P.M.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="43" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="44" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Notes:</string>
          </property>
         </widget>
        </item>
        <item row="45" column="0">
         <widget class="QTextEdit" name="tktNotes">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Is there any important information the techinician completeting the installation needs to know?</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>ticketInfo</zorder>
       <zorder>tktCalendar_D</zorder>
       <zorder>label_3</zorder>
       <zorder>tktDateText_D</zorder>
       <zorder>label_14</zorder>
       <zorder>tktNotes</zorder>
       <zorder>verticalSpacer_3</zorder>
       <zorder>tktCalendar_R</zorder>
       <zorder>label_2</zorder>
       <zorder>verticalSpacer_2</zorder>
       <zorder>tktDateText_R</zorder>
       <zorder>tkt_Group_TimeBlock</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>tktCalendar_D</tabstop>
  <tabstop>tktDateText_D</tabstop>
  <tabstop>tktCalendar_R</tabstop>
  <tabstop>tktDateText_R</tabstop>
  <tabstop>tkt_am</tabstop>
  <tabstop>tkt_pm</tabstop>
  <tabstop>tktNotes</tabstop>
  <tabstop>scrollArea_2</tabstop>
  <tabstop>cstAddress_D</tabstop>
  <tabstop>cstCity_D</tabstop>
  <tabstop>cstAddress_R</tabstop>
  <tabstop>cstCity_R</tabstop>
  <tabstop>tktTower</tabstop>
  <tabstop>tktQHolder</tabstop>
  <tabstop>tktQConfirm</tabstop>
  <tabstop>tktQDisclose</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>ticketTemplate</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>Copy</tabstop>
  <tabstop>Clear</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>tktCalendar_D</sender>
   <signal>clicked(QDate)</signal>
   <receiver>tktDateText_D</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tktDateText_D</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>tktCalendar_D</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tktCalendar_R</sender>
   <signal>clicked(QDate)</signal>
   <receiver>tktDateText_R</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>604</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>644</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tktDateText_R</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>tktCalendar_R</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
